app/views/posts/index.html.erb

```diff
@@ -2,10 +2,18 @@
 <%= render partial:"users/user", locals: {user: @user} %>
   <% @posts.each do |post| %>
     <%= render partial:"post", locals: {post: post} %>
-    <div class="container border border-black">
-      <% post.most_recent_comments.each do |comment| %>
-        <%= render partial:"comments/comment", locals: {comment: comment} %>
-      <% end %>
-    </div>
   <% end %>
+</div>
+<div class="d-flex justify-content-center">
+  <button type="button" class = "btn btn-dark mb-2 mt-2">
+      <%= link_to 'New Post', new_user_post_path %>
+  </button>
+<<<<<<< HEAD
+</div>
+<div class="d-flex justify-content-center">
+        <button type="button" class = "btn btn-dark mb-2 mt-2">
+            Pagination
+        </button>
+=======
```

Comment:
Please remove this after resolving the conflicts to make the code look clean 👍🏻

Reply:
Thanks for bringing that up, I fixed the issue concerning conflicts